### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by wandergithub**
🟢  Made the desktop version of portfolio using media queries to display correctly the website on screens 768px or wider.
🟢 lay out the elements using grid and flex box to replicate the template 4.
🟢 Adjusted background images to fit the new, responsive desktop version.
🟢 Solved all linter errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -59,17 +67,21 @@ <h2 class="title-2">Software Developer</h2>
...
         </nav>
-        <div>
+        <div id="work-section">
             <img src="sprites/Group55.png" alt="Down Arrow"><h2>Work</h2>
         </div>
         <h2 class="title">MY RECENT WORK</h2>
+        <h2 class="title desktop-v">My recent work</h2>
     </section>
     <img class="vector-img" src="sprites/Vector4.png" alt="End of section 3">
     <section id="section-4">
```
Comment (smunozmo) [inline index.html:87]: - [ ]  Your design is looking great in both versions, well done! However, we still need to invert every other card to look like the Figma template:

Template:
Captura de pantalla 2022-03-24 a la(s) 7 04 24 p  m

Your design:
Captura de pantalla 2022-03-24 a la(s) 7 04 42 p  m

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @wandergithub, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (amklass11) [reply index.html:87]: use a  row reverse attribute css

**Review by mikethreels — APPROVED** [review]
Hi Wander, 

Your project is complete! it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] [OPTIONAL] please make sure to use[ the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**